app/views/tracks/build/analyzer_tooltip.html.haml

```diff
@@ -0,0 +1,23 @@
+.c-tooltip-base
+  .flex.gap-16
+    = graphical_icon 'analyzer'
+    %div
+      %h5.text-h5
+        The #{@track.title} analyzer
+        = render ViewComponents::Track::Build::Health.new(@status.analyzer.health)
+
+      - health = @status.analyzer.health.to_sym
+      %p.text-p-base
+      - case health
+      - when :exemplar
+        Gives feedback on #{@status.analyzer.display_rate_percentage}% of the submitted iterations!
+      - when :healthy
+        Gives feedback on #{@status.analyzer.display_rate_percentage}% of the submitted iterations.
+      - when :needs_attention
+        No feedback has been given.
+      - when :missing
+        The analyzer has not yet been implemented.
```

Comment:
We can tweak this information later on, but this gives us something to work with (same for the other tooltips)